app/src/main/java/org/septa/android/app/MainActivity.java

```diff
@@ -138,15 +148,15 @@ public final void onCreate(@Nullable Bundle savedInstanceState) {
 
         // TODO: comment out when releasing to production
         // ShakeDetector initialization
-//        mSensorManager = (SensorManager) getSystemService(Context.SENSOR_SERVICE);
-//        mAccelerometer = mSensorManager.getDefaultSensor(Sensor.TYPE_ACCELEROMETER);
-//        mShakeDetector = new ShakeDetector();
-//        mShakeDetector.setOnShakeListener(new ShakeDetector.OnShakeListener() {
-//            @Override
-//            public void onShake(int count) {
-//                handleShakeEvent(count);
-//            }
-//        });
+        mSensorManager = (SensorManager) getSystemService(Context.SENSOR_SERVICE);
```

Comment:
Instead of commenting this in and out, maybe have the sensor manager creation determined by an external property, like a String value or something passed in via Gradle.  This way you can toggle it on and off more easily.